assets/src/design-system/components/menu/useDropDownMenu.js

```diff
@@ -73,9 +93,14 @@ export default function useDropDownMenu({
         focusedIndex < listLength - 1
       ) {
         handleMoveFocus(1);
+      } else if (
+        ['ArrowUp', isRTL ? 'ArrowRight' : 'ArrowLeft'].includes(key) &&
```

Comment:
should we make these a constant?

Reply:
I am neutral on it, it's not a pattern established but it's also a good idea! i'll add them to the constants in the menu